crates/relayer-utils/src/metric.rs

```diff
@@ -120,6 +136,9 @@ impl Metrics {
             "The Total number of data stored",
         );
 
+        let resource_metric_map: HashMap<ResourceId, ResourceMetric> =
+            HashMap::new();
+
```

Comment:
No types are needed here, rustc can infer that.

```suggestion
        let resource_metric_map = HashMap::new();
```